packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/getting-started.doc.ts

```diff
@@ -0,0 +1,140 @@
+- Embed [your app in Shopify POS](/docs/apps/pos/getting-started#embed-your-in-app-in-shopify-pos).
+      `,
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'step-1-generate-a-pos-ui-extension',
+      title: 'Step 1: Generate a POS UI Extension',
+      codeblock: {
+        title: 'Generate a POS UI Extension',
+        tabs: [
+          {code: `${examplePath}/generate-extension.bash`, language: 'bash'},
+        ],
+      },
+      sectionContent: `
+1. Navigate to your app directory.
+
+2. Ensure Shopify CLI is using the most up to date versions.
+
+3. Generate your POS UI extension template.
+
+4. Select \`POS UI\` under the Point-of-Sale menu.
+
+5. Give your extension a working name.
+
+6. Select the programming language that you want to work in.
+      `,
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'step-2-run-the-local-extension-in-your-development-store',
+      title: 'Step 2: Run the local extension in your development store',
+      codeblock: {
+        title: 'Install dependencies and start a local development server',
+        tabs: [
+          {
+            title: 'npm',
+            code: `${examplePath}/app-dev-npm.bash`,
+            language: 'bash',
+          },
+          {
+            title: 'yarn',
+            code: `${examplePath}/app-dev-yarn.bash`,
+            language: 'bash',
+          },
+        ],
+      },
+      sectionContent: `
+After you create your extension, you can [start a local development server](/docs/apps/getting-started/create#step-2-start-a-local-development-server) so that you can run your extension in your development store.
+
+When you start the server, Shopify CLI uses [Cloudflare](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/do-more-with-tunnels/trycloudflare/) to create a secure tunnel. Cloudflare gives your app extension a unique URL.
+
+1. Install your project's dependencies using the command from your package manager of choice.
+
+2. Start your local server for your extension in the app directory.
+
+3. Follow the prompts to associate your extension with your app and development store. Your development server should now be running.
+
+![Local Development Server Example](/assets/apps/pos/ui-ext-dev-server-console.png)
+
+To learn more about the processes that execute when you run \`dev\`, refer to the list of [Shopify CLI commands](/docs/api/shopify-cli/app/app-dev).
+      `,
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'step-3-install-your-app-and-preview-your-extension',
+      title: 'Step 3: Install your app and preview your extension',
+      sectionContent: `
+You can install your app and preview your extension in Shopify POS from the developer console.
+
+1. With your server running, press \`p\` to open the developer console.
+
+2. In the developer console page, complete the following tasks:
+
+  - To install your app, click on the preview link for the POS UI extension.
```

Comment:
I think we should distinguish between installing an app (which is already installed on the test store) and running an extension
```suggestion
  - To run your extension on POS, click on the preview link for the POS UI extension.
```